### DISCUSSION TITLE
Program not running for this reason, how can I fix this?

### DISCUSSION THREAD

**Discussion by brotatoechip927** [Avnsx/fansly#183] [General]
![error](https://github.com/Avnsx/fansly-downloader/assets/150559117/87a4d677-01e5-4bad-9108-43c893012fbe)

**Comment by brotatoechip927** [comment]
Sorry I'm a noob at this stuff, I don't understand what I need to do to fix this.

**Reply by melithine** [reply]
It's saying there's something malformed or missing in the `config.ini` file. Maybe post it here and we can point out what's wrong.

**Reply by melithine** [reply]
I should have suggested redacting your authorization token, that's a bad thing to post publicly. You'll want to delete or edit the image, possibly change your password. My apologies for not considering that.

I think the problem is line number 7 has a random entry of `undefined`... pretty sure the parser can't make any sense of that and thus it aborts.

**Reply by brotatoechip927** [reply]
o

**Reply by brotatoechip927** [reply]
All good, changed my password, I'll try removing the undefined and see if that does anything, thanks for the help

**Reply by brotatoechip927** [reply]
The program worked but now I get this error when trying to download a single post. 
![error2](https://github.com/Avnsx/fansly-downloader/assets/150559117/432ed7e0-7ef0-4932-b698-39a2f5a61302)

**Reply by brotatoechip927** [reply]
Are some content creators able to block downloads from this program maybe?

**Reply by brotatoechip927** [reply]
Sorry never mind, I just had the incorrect authorization code typed in, its working fine now. Thanks
